build_tools/kokoro/gcp_ubuntu/bazel/linux/x86-swiftshader/integrations/build_kokoro.sh

```diff
@@ -32,7 +32,7 @@ source "${KOKORO_ARTIFACTS_DIR?}/github/iree/build_tools/kokoro/gcp_ubuntu/docke
 docker_setup
 
 docker run "${DOCKER_RUN_ARGS[@]?}" \
-  gcr.io/iree-oss/bazel-tensorflow-swiftshader@sha256:269a3a42ca71fa6b040135056f1f5639c2f2c7d651099a88ddd513a011365144 \
+  gcr.io/iree-oss/bazel-tensorflow-swiftshader@sha256:4247a514aac61db114f615072acfc30f125895375b461d1a2b9127668bb18bb8 \
```

Comment:
Do we manually build&push `iree-benchmark-module` ?

Reply:
seems a rebase error. :(